staging/src/k8s.io/kubectl/pkg/cmd/create/create_configmap_test.go

```diff
@@ -33,11 +33,12 @@ func TestCreateConfigMap(t *testing.T) {
 		appendHash    bool
 		fromLiteral   []string
 		fromFile      []string
-		fromEnvFile   string
+		fromEnvFile   []string
 		setup         func(t *testing.T, configMapOptions *ConfigMapOptions) func()
 
-		expected  *corev1.ConfigMap
-		expectErr bool
+		expected     *corev1.ConfigMap
+		expectErr    bool
+		expectErrStr string
```

Comment:
I'd suggest consolidating into only the string field. We should never expect a blank error, so the boolean is kinda redundant and needs to be synchronized (e.g. right now you could set an error string but forget to set expectErr to true and it wouldn't get checked).

Reply:
makes sense